### DISCUSSION TITLE
Exporte log to external ELK stack

### DISCUSSION THREAD

**Discussion by leo-mercier** [telekom-security/tpotce#1040] [Q&A] [answered]
Hello, i have a project to make some T-POT honeypots with sensor mod and export all logs to an external ELK Stack. But in sensor mod their is no logstash. Do you have some advises ? Thank you

**Comment by t3chn0m4g3** [comment, marked as answer]
The next version of T-Pot will be released soon and will provide a distributed option to ship data into central ELK. You can review the code in the 22.x branch to get an idea how it works (docker/elk).

**Reply by leo-mercier** [reply]
Can you briefly tell me what i can change in my actual config (standard) to do it ? is it just the Host: in logstash configuration?

**Reply by t3chn0m4g3** [reply]
Check out the [Wiki](https://github.com/telekom-security/tpotce/wiki/Reconfigure-logstash.conf).

**Reply by leo-mercier** [reply]
thanks
